packages/engine/Source/Core/PolygonGeometry.js

```diff
@@ -1377,13 +1493,12 @@ PolygonGeometry.createGeometry = function (polygonGeometry) {
     return;
   }
 
```

Comment:
This looks like an existing issue, but the above call to `PolygonGeometryLibrary.polygonsFromHierarchy` (line 1368) is missing the `ellipsoid` argument. I'm not sure why this hasn't been a problem so far.

Reply:
Good catch. Likely, it doesn't come up that ellipsoid is not WGS84 in most cases, but it's best practice to pass it through.